lib/tasks/install-blueprint.js

```diff
@@ -80,6 +113,30 @@ class InstallBlueprintTask extends Task {
     return execa('npm', ['install'], { cwd });
   }
 
+  _npmInstallModule(module, cwd) {
+    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+    return execa('npm', ['install', module], { cwd })
```

Comment:
@rwjblue I would like to do something like that in a separate follow-up PR. as you can see a few lines above the existing code of this feature already uses npm exclusively.